tox.ini

```diff
@@ -284,6 +289,8 @@ commands_pre =
 
   pika: pip install {toxinidir}/instrumentation/opentelemetry-instrumentation-pika[test]
 
+  kafka: pip install {toxinidir}/instrumentation/opentelemetry-instrumentation-kafka[test]
```

Comment:
I would call it `opentelemetry-instrumentation-kafka-python` at it refers to the name of the package, not the framework